logstash/templates/statefulset.yaml

```diff
@@ -115,6 +115,15 @@ spec:
         podAffinity:
 {{ toYaml . | indent 10 }}
       {{- end }}
+      {{- if .Values.topologySpreadConstraints.enabled }}
+      topologySpreadConstraints:
+        - maxSkew: {{ .Values.topologySpreadConstraints.maxSkew }}
```

Comment:
@jgregmac this should be a loop to allow for more than one constraint.

Reply:
@cdenneen This recommendation has been implemented in my fork, although I do not see that change reflected in the "changed files" view of this PR.  The recent merge from master is reflected in the commit history though, so I am puzzled by that. Perhaps I need to file a new PR?

Reply:
https://github.com/elastic/helm-charts/blob/main/CONTRIBUTING.md#requirements-for-submiting-a-pull-request

Maybe new PR is necessary and looks like bumping Chart is necessary requirement too.